sktime/datatypes/_utilities.py

```diff
@@ -219,7 +271,7 @@ def update_data(X, X_new=None):
 ]
 
 
-def get_window(obj, window_length=None, lag=0):
+def get_window(obj, window_length=None, lag=None):
```

Comment:
please, update argument description in the docstring. specifically, default value

Reply:
not in the scope of this PR, but sure

Reply:
actually, I think the docstring was fine. The arg default is `None`, but this is interpreted as "0 of the right type". I added more explanation to the docstring to avoid user confusion. Is that ok?